### ISSUE TITLE
[Bug]: TPCH Q13 result is not correct 

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#8032]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):fb62c0e6960af6eedbdab06adecb2ebf7d97e8ba
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

TPCH Q10 expected result:
[q13.txt](https://github.com/matrixorigin/matrixone/files/10738644/q13.txt)

The actual result：
[tpch_q13_error.txt](https://github.com/matrixorigin/matrixone/files/10738683/tpch_q13_error.txt)

**Comment by aressu1985** [comment]
will test soon

**Comment by aressu1985** [comment]
fixed:6d4bd173514990032372310f7b3d9d803781074a
